modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -22,11 +22,7 @@
 sda.enkf.multisite <- function(settings, obs.mean, obs.cov, Q = NULL, restart=F, 
                                control=list(trace=T,
                                             FF=F,
-                                            interactivePlot=T,
-                                            TimeseriesPlot=T,
-                                            BiasPlot=F,
-                                            plot.title=NULL,
-                                            facet.plots=F,
+                                            plot=F,
                                             debug=FALSE,
                                             pause=F),
```

Comment:
Please use `TRUE`/`FALSE` throughout code like this. Again, this is primarily @para2x 's fault, but might as well fix it here while it's on our mind.

Reply:
I will try to get as many as possible. I have the SDA running on my computer right now so I don't want to mess with it until it finishes :)

Reply:
when you see certain R files pop up with the problem can you make a note for me. There are a lot of files for the SDA and I may not catch all them since I didn't make these ones. The call_MODIS one I can fix pretty easily since I made that

Reply:
fixed